lib/meaning-cloud.rb

```diff
@@ -4,7 +4,7 @@
 
 # Top level name space for the entire Gem.
 module MeaningCloud
-  API_BASE = 'https://api.meaningcloud.com/topics-1.2.php'
+  API_BASE = 'https://api.meaningcloud.com/topics-2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.